src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/requiredactions/RegisterRequiredActionAction.kt

```diff
@@ -23,7 +23,7 @@ class RegisterRequiredActionAction(
                         it.body().asReader(
                             Charset.defaultCharset()
                         ).readText()
-                    } status: ${it.status()}"
```

Comment:
Why are we removing the http -status here?

Reply:
probably merge issue. reverting this line